libgadget/utils/system.h

```diff
@@ -74,4 +74,9 @@ static inline int atomic_add_and_fetch(int * ptr, int value) {
     return k;
 }
 
+void MPIU_trace(MPI_Comm comm, int where, const char * fmt, ...);
+void MPIU_tracev(MPI_Comm comm, int where, const char * fmt, va_list va);
```

Comment:
These need some comments. Also, do we need to expose two functions for this?

Reply:
Yes. endrun calls the v version. just like sprintf / sprintfv.

Reply:
I added docstring in the C file.